.github/workflows/deploy-wheels-linux.yml

```
@@ -19,6 +19,7 @@ jobs:
             "manylinux2014_i686",
             "manylinux2014_x86_64",
             "musllinux_1_1_x86_64",
+            "musllinux_1_1_aarch64",
```

Let's stick them in alphabetical order, if speed or other reasons not applicable:

```suggestion
            "musllinux_1_1_aarch64",
            "musllinux_1_1_x86_64",
```

Plus I guess we'll need to add a PyPy3 exclusion (like line 35)?